server.js

```diff
@@ -1,9 +1,20 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const config = {
+	mongoURI:
+		'mongodb+srv://akshitp:E7lFFxThDg8fe7Dd@cluster0.6uq2j9y.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Use environment variables instead of hardcoding the MongoDB connection string. This is a better practice as it enhances security and flexibility. You can use the 'dotenv' package to load environment variables from a .env file. [important]